### ISSUE TITLE
Accessibility issue with message Wikimedia:Wikilearn-backend-e567d7-=7Bstart anchor=7DClick here=7Bend a/en

### ISSUE THREAD

**Issue by McDutchie** [wikimedia/edx-platform#436]
The current message text is:

```{start_anchor}Click here{end_anchor} to delete your single signed on (SSO) session.```

For accessibility, “click here” should never be a link text, because the link text should always describe the link. A more accessible version of this message would be:

```You can {start_anchor}delete your single signed on (SSO) session{end_anchor}.```

[Rationale from the W3C](https://www.w3.org/TR/WCAG20-TECHS/F84.html): *Many blind people who use screen readers call up a dialog box that has a list of links from the page. They use this list of links to decide where they will go. But if many of the links in that list simply say "click here" or "more" they will be unable to use this feature in their screen reader, which is a core navigation strategy. That's why it's a failure of 2.4.9 to not provide any way of allowing them to know the destination from the link text alone. It is also true for people who tab through links. If all they hear as they tab through the document is "click here, click here, click here etc." they will become confused.*

URL: https://translatewiki.net/w/i.php?title=Special:Translate&showMessage=wikilearn-backend-e567d7-%3D7Bstart_anchor%3D7DClick_here%3D7Bend_a&group=wikilearn-django&language=ia
